Alicerce/Sources/Logging/Chronicle/Log.swift

```diff
@@ -20,7 +20,7 @@ final class Log {
         return providers.count
     }
     
-    public class func registerProvider(provider: ProviderProtocol) {
+    public class func registerProvider(provider: LogProvider) {
```

Comment:
we could name this just `public class func register(provider: LogProvider)` what do you think? 🤔